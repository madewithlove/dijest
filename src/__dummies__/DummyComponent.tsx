import * as React from 'react';

const DummyComponent: React.SFC<{ text?: string }> = ({ text }) =>
    <div>
        <p>
            {text || 'foobar'}
        </p>
    </div>;

export default DummyComponent;
