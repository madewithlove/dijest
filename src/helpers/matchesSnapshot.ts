import toJson from 'enzyme-to-json';

export default function matchesSnapshot(wrapper) {
    expect(toJson(wrapper)).toMatchSnapshot();
}
