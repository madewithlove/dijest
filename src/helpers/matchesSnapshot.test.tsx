import * as React from "react";
import matchesSnapshot from "./matchesSnapshot";
import DummyComponent from "../__dummies__/DummyComponent";
import { mount } from "enzyme";

it("can snapshot a wrapper", () => {
    const wrapper = mount(<DummyComponent />);
    matchesSnapshot(wrapper);
});
