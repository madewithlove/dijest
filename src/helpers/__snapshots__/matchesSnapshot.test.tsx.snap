// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can snapshot a wrapper 1`] = `
<DummyComponent>
  <div>
    <p>
      foobar
    </p>
  </div>
</DummyComponent>
`;
